packages/ui-informed/src/form/Form.tsx

```diff
@@ -0,0 +1,83 @@
+import { cn } from '@utima/ui';
+import { Form as InformedForm } from 'informed';
+import { useMemo, type ComponentProps } from 'react';
+import { useTranslation } from 'react-i18next';
+import type { ZodObject, ZodRawShape } from 'zod';
+
+import { FormContext, type FormContextType } from '../hooks/useFormContext';
+import {
+  useFormSubmitActions,
+  type TypedFormState,
+} from '../hooks/useFormSubmitActions';
+
+export type FormProps<T> = Omit<
```

Comment:
can you rewrite all intersections (`&`) to interfaces with extend as it is in `utima/ui` (it's slight performance improvement in TSC but counts in a long run)